### PR TITLE
explicitly list binary dependencies

### PR DESCRIPTION
**Pull request by pkarman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,33 @@
 AudioMonster manipulates and transcodes audio.
 It wraps a number of different command line binaries such as sox, lame, flac, twolame, and ffmpeg.
 
+## Dependencies
+
+The following binary tools are required. They are available from most OS package managers. E.e. for OS X use homebrew:
```
Comment (kookster) [inline README.md]: [stickler mode on]
I think
"managers. E.e for OS X use homebrew:"

should read something like:
"managers, i.e. for OS X use homebrew:"
[stickler mode off]

**Comment by pkarman** [comment]
clarified my latin abbreviations.

**Comment by kookster** [comment]
pulchrum et bonum!

**Comment by annewootton** [comment]
plaudo!!!!!

guys just made my day

**Comment by pkarman** [comment]
as we say here in KS: ad astra per aspera (which is also a good local beer http://vimeo.com/41114016)

**Pull request merged**